Automattic/wp-calypso
# Playground: Replace jsx-to-string

[jsx-to-string](https://github.com/grommet/jsx-to-string) does not seem to be maintained. Switch to ~[pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format)~ [react-element-to-jsx-string](https://github.com/algolia/react-element-to-jsx-string) which is.

jsx-to-string has a number of dependencies (immutable, react, fbjs) that are out of sync with the rest of our dependencies. 

#### Testing instructions

* Load the playground in devdocs. Components should appear as they usually do.
* After `npm start` you should no longer see duplicate warnings for `react` and `fbjs`. There's still one for `immutable` but it should just be two versions instead of three.
